src/core/create_compilers/RollupCompiler.ts

```diff
@@ -41,18 +41,16 @@ export default function(files) {
 		var relative = ('' + href).substring(baseURI.length);
 		var link = document.querySelector('link[rel=stylesheet][href="' + relative + '"]')
 			|| document.querySelector('link[rel=stylesheet][href="' + href + '"]');
-		if (!link) {
+		if (link) {
+			fulfil();
+		} else {
 			link = document.createElement('link');
 			link.rel = 'stylesheet';
 			link.href = href;
+			link.onload = fulfil;
+			link.onerror = fulfil;
```

Comment:
I changed this behaviour slightly because it seems better to resolve the promise in the case that we can't load the style than just reject (which will make the application appear to hang).